packages/mdc-chips/_mixins.scss

```diff
@@ -438,6 +465,15 @@ $mdc-chip-ripple-target: ".mdc-chip__ripple";
     padding-right: $padding;
     padding-left: $padding;
   }
+
+  $margin: 0 - $padding;
+
+  #{$mdc-chip-ripple-target} {
+    @include mdc-feature-targets($feat-structure) {
+      margin-right: $margin;
```

Comment:
Can we remove negative margins from here too?

Reply:
Argh!!! had it removed from corresponding CL, but missed it on the PR.